boards/arm/nrf9131ek_nrf9131/doc/index.rst

```diff
@@ -0,0 +1,228 @@
+The nrf9131ek_nrf9131 board configuration provides support for the Nordic Semiconductor nRF9131 ARM
+Cortex-M33F CPU with ARMv8-M Security Extension and the following devices:
+
+* :abbr:`ADC (Analog to Digital Converter)`
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter with EasyDMA)`
+* :abbr:`WDT (Watchdog Timer)`
+* :abbr:`IDAU (Implementation Defined Attribution Unit)`
+
+.. figure:: img/nrf9131ek_nrf9131.jpg
+     :align: center
+     :alt: nRF9131 EK
+
+     nRF9131 EK (Credit: Nordic Semiconductor)
+
+The `Nordic Semiconductor Infocenter`_
+contains the processor's information and the datasheet.
+
+
+Hardware
+********
+
+nRF9131 EK has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 32 MHz.
+
+Supported Features
+==================
+
+The nrf9131ek_nrf9131 board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| SPU       | on-chip    | system protection    |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED (red) = P0.29
+* LED (green) = P0.30
+* LED (blue) = P0.31
```

Comment:
In `nrf9131ek_nrf9131_common.dtsi` these LEDs are labelled as "Green LED" 1-3. Perhaps it would be good to correct that.